ext/afform/gui/ang/afGuiEditor.js

```diff
@@ -62,6 +62,8 @@
               alert('Error: unknown form "' + $scope.afGuiEditor.name + '"');
             }
           }
+          $scope.canvasTab = 'layout';
```

Comment:
Strings placed directly in the root scope can't be manipulated by reference.
I'm probably going to update this module to use components or at least the `$scope.$ctrl` convention.

Reply:
Yeah, using a component would be a good change. The string-within-scope is less reliable (afaik, it's a risk that minor change might cause it to break).

Since it's currently a directive (ie there is no`$ctrl`), perhaps I leave it to use `$scope.canvasTab` (which parallels the existing `$scope.selectedEntityName`), and then the two can be changed at the same time?

Tangential thought: we do have a copy of https://angular-ui.github.io/bootstrap/ available. The nice thing about their tab widget is that (1) the controller/model binding is optional/if-needed and (2) the tab heading and tab body have stronger locality. But it would imply simpler markup (e.g. using `nav-tabs` without `panel-heading`). I [dabbled a little](https://github.com/civicrm/civicrm-core/commit/8c72f579d1eb39dd6ae40b06d6b9655341d02560) to see how that might look without the panel:

<img width="1102" alt="Screen Shot 2020-11-30 at 6 08 33 PM" src="https://user-images.githubusercontent.com/1336047/100689313-18f82480-3339-11eb-83d6-915e358501d1.png">